pkg/cmd/server/api/v1/types_test.go

```diff
@@ -118,6 +118,8 @@ imageConfig:
   format: ""
   latest: false
 kind: MasterConfig
+kubeAdmissionControlPluginsOverride:
```

Comment:
@deads2k re: the name... I think we should avoid using 'controller' in part of the name given that means something else in the code. It's really a list of plugins.